.github/workflows/tutorials.yml

```diff
@@ -7,28 +7,21 @@ on:
       - 'tutorials/*.*'
 
 
+env:
+  # Tutorials that require a GPU to run, so can't be run on CI without self-hosted runners
+  DONT_RUN: Tutorial2_ Tutorial9_ Tutorial13_ Tutorial18_
+
+
 jobs:
 
   run:
     runs-on: ubuntu-latest
 
     steps:
     - uses: actions/checkout@v2
-      with:
-        fetch-depth: 0
 
-    - run: echo "date=$(date +'%Y-%m-%d')" >> $GITHUB_ENV
-  
-    - name: Set up Python 3.7
-      uses: actions/setup-python@v2
-      with:
-        python-version: 3.7
-
-    - name: Cache Python
-      uses: actions/cache@v2
-      with:
-        path: ${{ env.pythonLocation }}
-        key: linux-${{ env.date }}-${{ hashFiles('**/setup.py') }}-${{ hashFiles('**/setup.cfg') }}-${{ hashFiles('**/pyproject.toml') }}
+    - name: Setup Python
+      uses: ./.github/actions/python_cache/
```

Comment:
In the PR description you say we won't cache it anymore, shouldn't we use `actions/setup-python@v2` then?

Reply:
I removed the caching from the nightly workflow. This is the workflow that run when the tutorials are changed, so here caching makes sense... but it was still done the old way, so not working :woman_facepalming: